rialto_airflow/dags/harvest.py

```diff
@@ -21,10 +21,13 @@
 sul_pub_key = Variable.get("sul_pub_key")
 
 # to artificially limit the API activity in development
+dev_limit = None
 try:
     dev_limit = int(Variable.get("dev_limit", default_var=None))
 except TypeError:
-    dev_limit = None
+    pass
+except ValueError:
+    pass
```

Comment:
fwiw, i got a `ValueError` (tried to call `int` on `''` before i set the variable, not sure if `None` was turned to that or what) when starting airflow locally to confirm it still worked.  since the spirit of this seemed to be resilience against an undeclared variable, this seemed like the right thing to do?